xonsh/prompt/base.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os
+import socket
+
+from xonsh.lazyasd import LazyObject
+from xonsh.tools import is_superuser
+from xonsh.platform import ON_WINDOWS
```

Comment:
You should not do `from ... import` for modules outside of the subpackage.  This kinds of imports cannot be performed lazily.